packages/eslint-plugin/rules/max-statements-per-line/max-statements-per-line._js_.ts

```diff
@@ -174,5 +181,13 @@ export default createRule<RuleOptions, MessageIds>({
       'ExportAllDeclaration:exit': leaveStatement,
       'Program:exit': reportFirstExtraStatementAndClear,
     }
+
+    for (const key in listeners) {
+      if (ignoredNodes.includes(key) || ignoredNodes.includes(`${key}:exit`)) {
+        delete listeners[key]
```

Comment:
Maybe we should have a string array to list all the enter statements, and list them as enum in the config, and then construct the listener instead of removing them. For better UX and more robustness.

Reply:
I'm not sure if I understand what you mean, is it like I changed it?

Reply:
Yes, the direction is what I was imagining. But I think instead of of `listeningNodes` I think it would be easier to use `ignoredNodes` to exclude

Reply:
oh, that's what you mean. this is indeed much better. thank you for your help